code/go/0chain.net/smartcontract/dbs/event/mint.go

```diff
@@ -0,0 +1,10 @@
+package event
+
+type Mint struct {
```

Comment:
I think we can make do with replacing this table with a field in the `block` table. We only want the total amount minted each block, not a list of mints per block.

Reply:
This is not a table... it is just a mint structure... which we use to add mint to block table